test/python/transpiler/test_decompose.py

```diff
@@ -79,8 +79,6 @@ def test_decompose_none(self):
         circuit.h(qr[0])
         dag = circuit_to_dag(circuit)
         pass_ = Decompose(HGate)
```

Comment:
```suggestion
        pass_ = Decompose()
```

The name of this test is `test_decompose_none`.  I suspect it should be changed to `pass_ = Decompose()`, otherwise it's identical to the test above.